### PR TITLE
Awards Clusters

### PR DESCRIPTION
**Pull request by bandildo**
Implements #852 

Now the .xml supports the following format:

```
	<award>
		<name>Award Name</name>
		<description>Descriptions.</description>
		<medal>Medal1.png</medal>
		<medal>Medal2.png</medal>
		<medal>Medal3.png</medal>
		<ribbon>Ribbon1.png</ribbon>
		<ribbon>Ribbon2.png</ribbon>
		<ribbon>Ribbon3.png</ribbon>
		<misc>Misc1.png</ribbon>
		<misc>Misc2.png</ribbon>
		<xp>3</xp>
		<stackable>true</stackable>
	</award>
```

So, when you give the first award, it will display Medal1.png, Ribbon1.png and Misc1.png. If you award it again, it will display Medal2.png, Ribbon2.png and Misc2.png.

The order by which each image is displayed is the same order it is in the .xml. There is no need to include the number in the end of the filename.

When the limit is reached, the last one will be displayed. In this example, after awarding this the 3rd time, the misc will still be Misc2.png.

These changes are backward compatible with the save-games that used the first award implementation.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/personnel/AwardsFactory.java [hunk 1]
```diff
@@ -143,30 +145,36 @@ private void loadAwards(){
...
-                }
-                awardsMap.put(currentSetName, tempAwardMap);
-            } catch (SAXException | ParserConfigurationException e) {
-                MekHQ.getLogger().error(AwardsFactory.class, "loadAwards", e);
-            } catch (FileNotFoundException e) {
-                MekHQ.getLogger().error(AwardsFactory.class, "loadAwards", "Cannot find XML awards file: " + file.getName(), e);
-            } catch (JAXBException | IOException e) {
-                MekHQ.getLogger().error(AwardsFactory.class, "loadAwards", "Error loading XML for awards", e);
+            Map<String, Award> tempAwardMap = new HashMap<>();
+            String currentSetName = fileName.replaceFirst("[.][^.]+$", "");
```
Comment (sixlettervariables) [inline MekHQ/src/mekhq/campaign/personnel/AwardsFactory.java]: Are you trying to strip the extension from the file name?

Comment (bandildo) [reply MekHQ/src/mekhq/campaign/personnel/AwardsFactory.java]: Yessir.

**Comment by bandildo** [comment]
I've fixed the merging issues.

However, out of nowhere, I'm having a consistent graphical glitch:

image

Am I the only one?

**Comment by bandildo** [comment]
I've fixed some issues and should be ready to merge, fixing #852 and #848

**Comment by bandildo** [comment]
Solved latest merge conflicts, ready to go.

**Pull request merged**